src/issue/issue.action.js

```diff
@@ -242,3 +244,36 @@ export const mergePullRequest = (
       });
   };
 };
+
+export const submitNewIssue = (
+  owner,
+  repo,
+  issueTitle,
+  issueComment
+) => {
+  return (dispatch, getState) => {
+    const accessToken = getState().auth.accessToken;
+
+    dispatch({ type: SUBMIT_NEW_ISSUE.PENDING });
+
+    return fetchSubmitNewIssue(
+      owner,
+      repo,
+      issueTitle,
+      issueComment,
+      accessToken
+    )
+      .then(data => {
+        dispatch({
```

Comment:
This is where you can potentially navigate to the issue screen with the `issue` response :)